src/style.css

```diff
@@ -0,0 +1,79 @@
+* {
+  box-sizing: border-box;
+}
+
+.page {
+  font-family: "Roboto", sans-serif;
+}
+
+.stars_section {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 24px;
+  margin-top: 16px;
+}
+
+.stars {
+  display: flex;
+  flex-direction: row;
+}
+
+.star {
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+  background: url("./images/star.svg") no-repeat center;
+}
+
+.stars--1 .star:nth-child(-n+1),
+.stars--2 .star:nth-child(-n+2),
+.stars--3 .star:nth-child(-n+3),
+.stars--4 .star:nth-child(-n+4),
+.stars--5 .star:nth-child(n) {
+  background-image: url("./images/star-active.svg");
+}
+
+.stars_reviews {
+  display: flex;
+  align-items: flex-end;
+  font-size: 10px;
+  line-height: 14px;
+  color: #060b35;
+}
+
+.price {
```

Comment:
`price` shouldn't be a separate BEM block, make it the element
You should have only 2 bem blocks: for the card and for the stars